docs/kratos/manage-identities/30_account-recovery.mdx

```diff
@@ -9,42 +9,39 @@ import TabItem from "@theme/TabItem"
 import CodeBlock from "@theme/CodeBlock"
 import SelfHostedKratos from "@site/src/components/SelfHostedKratos"
 
-You can initiate account recovery for identities using the admin API endpoints.
-
-For more detailed information and general guidelines on account recovery, take a look at the
-[account recovery and password reset](../self-service/flows/account-recovery-password-reset.mdx) section.
+You can initiate account recovery for users using the admin API endpoints. You can initiate the flow even for users that don't
+have a recovery address configured.
 
 :::note
 
-You can initiate account recovery for a user without a recovery address via the admin API, but if the recovery flow expires the
-users can't re-initiate the flow by themselves.
+If the recovery flow initiated through the admin API expires, users without a recovery address can't start the flow again by
+themselves.
 
 :::
 
-## One time codes
+Read [this document](../self-service/flows/account-recovery-password-reset) to learn more about the account recovery flow.
+
+## One-time codes
+
+````mdx-code-block
+<Tabs>
+<TabItem value="oc" label="Ory Network" default>
 
-<Tabs
-  defaultValue="curl"
-  values={[
-    {label: 'curl', value: 'curl'},
-    {label: 'GoLang', value: 'go'},
-    {label: 'Self-Hosted Ory Kratos', value: 'kratos'},
-  ]}>
-  <TabItem value="curl">
+Send a request to the admin API of your project:
 
 ```shell
 curl --request POST -sL \
-  --header "Authorization: Bearer ory_pat_xRKLsFEOUFQFVBjd6o3FQDifaLYhabGd" \
+  --header "Authorization: Bearer ORY_PAT" \
   --header "Content-Type: application/json" \
   --request POST \
   --data '{
   "expires_in": "12h",
   "identity_id": "e01b5f2f-6afc-4194-8578-4cebcf69a4d5"
-}' https://playground.projects.oryapis.com/admin/recovery/code
+}' https://{your-project-slug}.projects.oryapis.com/admin/recovery/code
```

Comment:
It's not really a "quickstart" kind of thing (this document, I mean) so I personally feel that it's better to make users work with their own projects rather than the playground.

Reply:
Also, just something I noticed when reading the docs, but if you can't "copy & paste" it than we might as well leave it out, because no one will go through the trouble of adjusting it. :) But again, this is not blocking, just something we should find a solution for in the future.